README.md

```diff
@@ -1,3 +1,62 @@
 # git-hooks
 [![](https://github.com/unthreaded/git-hooks/workflows/Git%20hooks%20Pipeline/badge.svg)](https://github.com/unthreaded/git-hooks/actions?workflow=Git+hooks+Pipeline)
-[![codecov](https://codecov.io/gh/unthreaded/git-hooks/branch/master/graph/badge.svg)](https://codecov.io/gh/unthreaded/git-hooks)
\ No newline at end of file
+&nbsp;&nbsp;[![codecov](https://codecov.io/gh/unthreaded/git-hooks/branch/master/graph/badge.svg)](https://codecov.io/gh/unthreaded/git-hooks)
+
+## What is git-hooks?
+Git hooks is a commit hook that works with git to make sure your issue, ticket, Jira, and story numbers make it to the commit message.
+
+For example:
+```shell script
+git checkout -b feature/GH-123-example-branch
+...
+# stage some work
+...
+git commit -m "Completed work and unit tests"
+...
+git log -n 1 --format=oneline
+afb126992c7e780939ef9b931f38c2cb0c47f91f (HEAD -> feature/GH-123-example-branch) GH-123: Completed work and unit tests
+```
+Notice that the issue number, GH-123, was copied into the commit message.
+
+Our hook can even handle the lack of a ticket:
+```shell script
+git checkout -b need-this-yesterday
+...
+git commit -m "PROD FIX"
+...
+git log -n 1 --format=oneline
+afb126992c7e780939ef9b931f38c2cb0c47f91f (HEAD -> feature/GH-123-example-branch) GH-123: Completed work and unit tests
+```
```

Comment:
wrong example perhaps? shouldn't the result of `git commit -m "PROD FIX"` in branch `need-this-yesterday` be `NOGH: PROD FIX`